docs/user/config/astTransformers.md

```diff
@@ -23,11 +23,17 @@ module.exports = {
   globals: {
     'ts-jest': {
       astTransformers: {
-        before: ['my-custom-transformer'],
```

Comment:
the documentation should have 2 separate sections, one for defining transformers with as `string[]` and one for `AstTransformer[]`